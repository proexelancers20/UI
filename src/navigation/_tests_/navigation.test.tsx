import React from 'react';
import Navigation from '..';
import {render} from '@testing-library/react-native';

jest.mock('@react-navigation/native-stack', () => ({
  createNativeStackNavigator: () => ({
    Navigator: 'mocked-Navigator',
    Screen: 'mocked-Screen',
  }),
}));
jest.mock('@react-navigation/native', () => ({
  NavigationContainer: jest.fn(),
}));
jest.mock('react-native-size-matters',() => ({
    scale: jest.fn(), 
    verticalScale: jest.fn()
}));

const component = <Navigation />;

describe('Render Navigation', () => {
  test('snap', () => {
    const container = render(component);
    expect(container.toJSON()).toMatchSnapshot();
  });
});
