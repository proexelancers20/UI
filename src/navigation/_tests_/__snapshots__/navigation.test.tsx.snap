// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Navigation snap 1`] = `null`;
